VoteSystem.Console/Program.cs

```diff
@@ -114,53 +114,63 @@ static void Main(string[] args)
                                 userInterface.ShowPollsConsole();
                                 Console.WriteLine("Choose the poll:");
                                 string poll_temp_name = Console.ReadLine();
-                                Poll poll1 = pollRepository.Get(poll_temp_name);
-                                if (pollService.CheckAllPolicy(poll_temp_name, user_temp_id).Count > 0)
+                                var pollPolicyFailedChecks = pollService.CheckAllPolicy(poll_temp_name);
+                                if (pollPolicyFailedChecks.Count > 0)
                                 {
-                                    foreach (var a in pollService.CheckAllPolicy(poll_temp_name, user_temp_id))
+                                    foreach (var a in pollPolicyFailedChecks)
                                     {
                                         Console.WriteLine(a.Value);
                                         Console.ReadLine();
                                     }
                                     break;
                                 }
-                                foreach (var a in pollRepository.GetChoices(poll_temp_name))
+
+                                Poll poll1 = pollRepository.Get(poll_temp_name);
```

Comment:
and now, given aggregate id "properly returned from reposuitory" - you no longer need to `pollRepository.GetChoices`